src/subgraph/api.ts

```diff
@@ -18,6 +18,7 @@ export const SUBGRAPH_PROD_CONFIG: ApiBaseUrls = {
   [SupportedChainId.MAINNET]: SUBGRAPH_BASE_URL + '/cow',
   [SupportedChainId.GNOSIS_CHAIN]: SUBGRAPH_BASE_URL + '/cow-gc',
   [SupportedChainId.GOERLI]: SUBGRAPH_BASE_URL + '/cow-goerli',
+  [SupportedChainId.SEPOLIA]: SUBGRAPH_BASE_URL + '/cow-sepolia',
```

Comment:
I haven't tests subgraph endpoints, but I think it's not so critical now

Reply:
Can it not be added then?
I rather wait until we have the subgraph deployed to not vendor the sdk with non-working changes.

Reply:
I can keep it empty

Reply:
Done